test/api/it.spec.js

```diff
@@ -374,5 +387,45 @@ describe('expect.it', () => {
         'expect.it(<function>) does not accept additional arguments'
       );
     });
+
+    describe('and chained the expression is chained', () => {
+      describe('and the first expression is a function', () => {
+        it('fails with a diff including all items in the chain', () => {
+          expect(
+            function() {
```

Comment:
Should be fine to use arrow functions here, as long as they're not part of the expected error message.